docs/first-order/receptive_fields.ipynb

```diff
@@ -23,7 +23,7 @@
   },
```

Comment:
<p>the first two can be made more brief. can probably replace the above comment I wrote then, and for the first one could say "creates dict that maps channel ids to brain location" and for the second one, "creates dict that maps channel ids to probe group name"</p><p> </p><p>good comments for functions, probably pretty helpful. Can remove the words 'function retrieves __' since you already mention what it returns and the function name indicates this too. also typos</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AllenInstitute/openscope_databook/pull/237/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>